### DISCUSSION TITLE
Missing K Factor calibration in Orca

### DISCUSSION THREAD

**Discussion by BeGoneBaizuo** [OrcaSlicer/OrcaSlicer#1291] [General]
I don't know what I'm missing. However, when I go into the device tab the calibrate option isn't there. It's missing in both Bambu slicer and Orca. I have a KS X1C and it has never been able to print on anything except the cool plate. Any PEI, whether textured or smooth has horrible adhesion and comes loose. I've tried every temperature range possible for both bed and filament. I've dried and changed filament brands. Bambu just says to slow the print down, but that doesn't work. After searching the discord and reddit I see many others have had this same issue, and changing the K Factor has helped some. Any help would be appreciated. Attached is a snip of my device tab in Orca. 
![ORCA MISSING K](https://github.com/SoftFever/OrcaSlicer/assets/116326334/a441c216-85fe-4a02-8cda-a8072492adcf)

**Comment by julie777** [comment]
@BeGoneBaizuo You say the calibration option is not there, but I see this in the image you added.
![image](https://github.com/SoftFever/OrcaSlicer/assets/5090005/1f2c3c58-3dd6-4615-a078-41207e574de9)

Is there another calibration? (I am just trying to learn more.)

**Comment by LJKraus** [comment]
It's an old question and hopefully @BeGoneBaizuo BeGoneBaizuo has already found the answer, but...

I think the difference in terminology is causing confusion.
K-factor (also known as linear advance) is called Pressure Advance in Orca Slicer.  
Calibrate here: 
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/3bf506ab-094f-4497-8f35-e2de86a19889)

Then save the result in the filament settings:
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/dcffe241-24ee-4fc1-acaa-af9c380bcd08)

**Comment by BeGoneBaizuo** [comment]
Hello,

I did figure it out. The confusion came from a bambulab wiki someone had
posted.
